### DISCUSSION TITLE
How to change an edge kind once created?

### DISCUSSION THREAD

**Discussion by ChildishGiant** [gephi/gephi#2517] [Q&A] [answered]
Is there any way to change the kind of edge between two nodes once it exists? I've tried double clicking in the data table like you can for the label but no luck.

**Comment by mbastian** [comment, marked as answer]
Unfortunately no, but this is a legitimate request that we should implement. I added an issue so we don't forget about it: https://github.com/gephi/gephi/issues/2518
